pkg/jobs/update.go

```diff
@@ -348,15 +348,6 @@ func (ju *JobUpdater) hasUpdates() bool {
 	return ju.md != JobMetadata{}
 }
 
-// UpdateRunStats is used to update the exponential-backoff parameters last_run and
-// num_runs in system.jobs table.
-func (ju *JobUpdater) UpdateRunStats(numRuns int, lastRun time.Time) {
-	ju.md.RunStats = &RunStats{
-		NumRuns: numRuns,
-		LastRun: lastRun,
-	}
-}
```

Comment:
If we completely remove this, do we also want to remove the code that populates this on the JobMetadata (since we aren't setting it anywhere) and the code in update that looks for whether it is set and adds it to the update statement?

Reply:
Removed it too now.